jquantsapi/client.py

```diff
@@ -6,7 +6,11 @@
 import requests
 from dateutil import tz
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry  # type: ignore
+
+try:
+    from requests.packages.urllib3.util.retry import Retry  # type: ignore
+except ImportError:
+    from urllib3.util import Retry
```

Comment:
これ始めから `from urllib3.util import Retry` でもよかったりでしょうか？

```suggestion
from urllib3.util import Retry
```

Reply:
可能ですー、urllib3.util Retryからはmethod_whitelistのパラメータがいずれdeprecateされるとか書いててちょっと不安だったのであたらしいrequestsバージョン使ってる人にはrequests.packages.urllib3.util.retryの方使わせるかとか思ったけど現時点ではどちらでも動くと思います

Reply:
なるほど！互換性のために取り込んだ可能性があるのですね。